sannytee/flight-booking-api
# #166076279 build the model for the application

#### What does this PR do?
This PR set up the model and database for the application

#### Description of Task to be completed?
- create the User and flight model

### How should this be manually tested?
- Pull the branch to your local machine

- check the env.sample file for required environment variables and source them.

- run `python manage.py migrate`  to migrate models into the database

- check the database to confirm the changes.

#### What are the relevant pivotal tracker stories?
[#166076279](https://www.pivotaltracker.com/story/show/166076286)
